client/src/app/EmptyTab.less

```diff
@@ -62,6 +62,10 @@
       font-weight: 14px;
     }
 
+    p {
```

Comment:
Did you know what (external) styles interfer here? Adding these styles feels like a hack to me, if we don't find the cause of the original blunders (some extension does not apply styles locally).

Reply:
I think it comes from style in `variable-outline`. Investigating... 👀

https://github.com/bpmn-io/variable-outline/blob/31340baaf9c7a9e0b5a5bb0205a6205aec29a973/lib/style.scss

Reply:
I think so, too. But either way, let's fix it at the root, not the leaf.

Reply:
Should we still merge the rest of the changes? I think it's ok to have an explicit style for empty (welcome) tab header.

Reply:
What is the benefit of adding it, if it worked previously without that change?